------------------------------------------------------------------------
r44518 | kakuto | 2017-06-07 04:53:28 -0400 (Wed, 07 Jun 2017) | 1 line

luatexdir/lua/liolibext.c; Fix typos which lead to a bug in io.popen() pointed out by gregorio team

Index: Build/source/texk/web2c/luatexdir/lua/liolibext.c
===================================================================
--- Build/source/texk/web2c/luatexdir/lua/liolibext.c	(revision 44517)
+++ Build/source/texk/web2c/luatexdir/lua/liolibext.c	(revision 44518)
@@ -365,7 +365,7 @@
         lua_pushliteral(L,"all command execution is disabled");
     } else if (restrictedshell == 0) {
         lua_pushboolean(L,1);
-        lua_pushliteral(L,"all commands are permitted");
+        lua_pushstring(L,filename);
     } else {
         char *safecmd = NULL;
         char *cmdname = NULL;
@@ -374,10 +374,6 @@
                 lua_pushboolean(L,0);
                 lua_pushliteral(L, "specific command execution disabled");
                 break;
-            case 1:
-                lua_pushboolean(L,1);
-                lua_pushstring(L,filename);
-                break;
             case 2:
                 lua_pushboolean(L,1);
                 lua_pushstring(L,safecmd);

------------------------------------------------------------------------
